core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -736,11 +738,22 @@ private CarbonCommonConstants() {
   @CarbonProperty(dynamicConfigurable = true)
   public static final String CARBON_MAJOR_COMPACTION_SIZE = "carbon.major.compaction.size";
 
+  /**
+   * Size of Minor Compaction in MBs
+   */
+  @CarbonProperty(dynamicConfigurable = true)
```

Comment:
Just the table property is enough, why again a carbon property is needed ?

Reply:
@ajantha-bhat some users need system value to control all tables if use same value, like major compaction system value. I think by default we dont use it, and the user can specify for each table, if he want to set for all tables, he can use the system level parameter.